references/detection/train_pytorch.py

```diff
@@ -112,7 +112,6 @@ def main(args):
         img_folder=os.path.join(args.val_path, 'images'),
         label_path=os.path.join(args.val_path, 'labels.json'),
         sample_transforms=T.Resize((args.input_size, args.input_size)),
-        rotated_bbox=args.rotation
```

Comment:
Mmmh I have the feeling it's not doing the same thing.
Passing it here will output rotated bbox (source data being a polygon), but with your modification, only the model will be allowed to produce bboxes. Shouldn't we keep this?

Reply:
If we pass this arg, the dataset will try to fit a rotated box from the targets, which will throw an error when we use sample_transform for the rotation transformation which takes only straight boxes as inputs.

Reply:
Oh I see, but then the solution should be to make special cases for rotated bbox in the transformations in the long-term! For now, I agree, let's keep things simple :ok_hand: